src/services/account/useInvoices.ts

```diff
@@ -0,0 +1,150 @@
+import { useQuery } from '@tanstack/react-query'
+import { z } from 'zod'
+
+import Api from 'shared/api'
+import { NetworkErrorObject } from 'shared/api/helpers'
+
+const DefaultPaymentMethodSchema = z
+  .object({
+    billingDetails: z.object({
+      address: z.object({
+        city: z.string().nullable(),
+        country: z.string().nullable(),
+        line1: z.string().nullable(),
+        line2: z.string().nullable(),
+        postalCode: z.string().nullable(),
+        state: z.string().nullable(),
+      }),
+      email: z.string().nullable(),
+      name: z.string().nullable(),
+      phone: z.string().nullable(),
+    }),
+    card: z.object({
+      brand: z.string().nullable(),
+      expMonth: z.number().nullable(),
+      expYear: z.number().nullable(),
+      last4: z.string().nullable(),
+    }),
+  })
+  .nullable()
+
+const InvoiceSchema = z.object({
+  amountDue: z.number(),
+  amountPaid: z.number(),
+  created: z.number(),
+  currency: z.string().nullable(),
+  customerAddress: z.string().nullable(),
+  customerEmail: z.string().nullable(),
+  customerName: z.string().nullable(),
+  defaultPaymentMethod: DefaultPaymentMethodSchema,
+  dueDate: z.number().nullish(),
+  footer: z.string().nullable(),
+  id: z.string().nullable(),
+  lineItems: z
+    .array(
+      z.object({
+        amount: z.number().nullable(),
+        currency: z.string().nullable(),
+        description: z.string().nullable(),
+      })
+    )
+    .nullable(),
+  number: z.string().nullable(),
+  periodEnd: z.number().nullable(),
+  periodStart: z.number().nullable(),
+  status: z.string().nullable(),
+  subtotal: z.number().nullable(),
```

Comment:
not nullable

Reply:
Fixed in [d3feda3](https://github.com/codecov/gazebo/pull/2920/commits/d3feda35e6c8b035ff9b97a4900df603949e6c94)!